README.md

```diff
@@ -18,4 +18,10 @@ For instance, [Stanford bunny LICENSE](points/bunny/LICENSE) applies to [Stanfor
 | [Stanford bunny](points/bunny/bunny.off) | [Stanford University](http://graphics.stanford.edu/data/3Dscanrep/) | [OFF file](https://en.wikipedia.org/wiki/OFF_(file_format)) with 35947 points in $\mathbb{R}^3$ and 69451 triangles |
 | [Spiral 2D](points/spiral_2d/spiral_2d.csv) | Generated within GUDHI | CSV file with 114562 points in $\mathbb{R}^2$ |
 | [Daily and sports activities](points/activities/activities_p1_left_leg.csv) | [UC Irvine ML repository](https://archive.ics.uci.edu/ml/datasets/daily+and+sports+activities) - [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/legalcode) | CSV file with 30000 points in 3d + activity type column |
-| [SO3](points/SO3/SO3_50000.off) | The SO3 points clouds have been generated with [the ISOI software](https://mitchell-web.ornl.gov/SOI/index.php) | [OFF file](https://en.wikipedia.org/wiki/OFF_(file_format)) with 50000 points in $\mathbb{R}^9$ |
\ No newline at end of file
+| [SO3](points/SO3/SO3_50000.off) | The SO3 points clouds have been generated with [the ISOI software](https://mitchell-web.ornl.gov/SOI/index.php) | [OFF file](https://en.wikipedia.org/wiki/OFF_(file_format)) with 50000 points in $\mathbb{R}^9$ |
+
+## Time Series
+
+| Model Name | Source | Description                                                                                                                                                                           |
+|------------|--------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Topological Wheels](timeseries/topological-wheels/tw-data-annotated.png) | [DataShape](https://www.inria.fr/fr/datashape) | 10 CSV files where each file has 64 timeseries over 10000 timestamps each, and a $y$ ground truth indicating anomalous regime. See [README](timeseries/topological-wheels/README.md). |
```

Comment:
I have noticed the links were not correct (`_` and not `-` in topological_wheels directory)
```suggestion
| [Topological Wheels](timeseries/topological_wheels/tw-data-annotated.png) | [DataShape](https://www.inria.fr/fr/datashape) | 10 CSV files where each file has 64 timeseries over 10000 timestamps each, and a $y$ ground truth indicating anomalous regime. See [README](timeseries/topological_wheels/README.md). |
```